Source/FolioReaderCenter.swift

```diff
@@ -70,12 +71,19 @@ open class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UIColl
     fileprivate var currentWebViewScrollPositions = [Int: CGPoint]()
     fileprivate var currentOrientation: UIInterfaceOrientation?
 
+    fileprivate var readerConfig: FolioReaderConfig {
+        guard let readerContainer = readerContainer else { return FolioReaderConfig() }
+        return readerContainer.readerConfig
+    }
+
     fileprivate var book: FRBook {
-        return self.readerContainer.book
+        guard let readerContainer = readerContainer else { return FRBook() }
+        return readerContainer.book
     }
 
-    fileprivate var readerConfig: FolioReaderConfig {
-        return self.readerContainer.readerConfig
+    fileprivate var folioReader: FolioReader {
+        guard let readerContainer = readerContainer else { return FolioReader() }
+        return self.readerContainer!.folioReader
```

Comment:
like if you had kept the non-optional value this would not have been needed.
Plus you should use the value from the `guard` instead of the `self.readerContainer!.folioReader`